ycms/cms/views/zentral.py

```diff
@@ -0,0 +1,51 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from ..models import Ward , Patient, User
+from django.template import loader
+from django.shortcuts import render
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
+from django.views.generic import TemplateView
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+    template_name = "zentral.html"
+
+    def newward(self, request):
+        """
+        This function returns a list of all wards,
+        as well as a form for creating a new one(not implemented yet)
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        """
+        #ward list which acts as a iterable object
+        total_wards = Ward.objects.all()
+        #total number of wards
+        total_wards_count = Ward.objects.all().count()
+        #total number of beds
+        total_beds_count = wards_count*28
```

Comment:
```suggestion
        total_beds_count = total_wards_count*28
```

However, this is not accurate. Rooms can have more than 2 beds, and wards can have more or fewer than 14 rooms. What you are looking for is
```python
total_beds_count = sum(ward.total_beds for ward in Ward.objects.all())
```
which uses the `total_beds` cached property we already implemented on the Ward model.